server.js

```diff
@@ -1311,29 +1311,36 @@ cache(function(data, match, sendBadge, request) {
 // Bountysource integration.
 camp.route(/^\/bountysource\/team\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var team = match[1];  // eg, `mozilla-core`.
-  var type = match[2];  // eg, `activity`.
-  var format = match[3];
-  var url = 'https://api.bountysource.com/teams/' + team;
-  var options = {
+  const team = match[1];  // eg, `mozilla-core`.
+  const type = match[2];  // eg, `activity`.
+  const format = match[3];
+  const url = 'https://api.bountysource.com/teams/' + team;
+  const options = {
     headers: { 'Accept': 'application/vnd.bountysource+json; version=2' } };
-  var badgeData = getBadgeData('bounties', data);
+  let badgeData = getBadgeData('bounties', data);
   request(url, options, function dealWithData(err, res, buffer) {
     if (err != null) {
       badgeData.text[1] = 'inaccessible';
       sendBadge(format, badgeData);
       return;
     }
     try {
-      var data = JSON.parse(buffer);
+      if (res.statusCode !== 200) {
+        throw Error('Bad response.');
+      }
+      const data = JSON.parse(buffer);
```

Comment:
I know this is also done in other badges, but a parameter called `data` is already passed in. Maybe you should rename it to `parsedData` instead to avoid confusion and reduce the risk of future coding mistakes.